recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.7" %}
+{% set version = "0.8-RC2" %}
```

Comment:
@CJ-Wright looks like `RC2` in capital letters is passing the bot filter :-/

Reply:
Hmm ok